datalake/src/main/java/com/sequenceiq/datalake/flow/datalake/cmsync/SdxCmSyncFlowConfig.java

```diff
@@ -0,0 +1,68 @@
+package com.sequenceiq.datalake.flow.datalake.cmsync;
+
+
+import static com.sequenceiq.datalake.flow.datalake.cmsync.SdxCmSyncEvent.SDX_CM_SYNC_FAILED_EVENT;
+import static com.sequenceiq.datalake.flow.datalake.cmsync.SdxCmSyncEvent.SDX_CM_SYNC_FAILED_HANDLED_EVENT;
+import static com.sequenceiq.datalake.flow.datalake.cmsync.SdxCmSyncEvent.SDX_CM_SYNC_FINALIZED_EVENT;
+import static com.sequenceiq.datalake.flow.datalake.cmsync.SdxCmSyncEvent.SDX_CM_SYNC_FINISHED_EVENT;
+import static com.sequenceiq.datalake.flow.datalake.cmsync.SdxCmSyncEvent.SDX_CM_SYNC_IN_PROGRESS_EVENT;
+import static com.sequenceiq.datalake.flow.datalake.cmsync.SdxCmSyncEvent.SDX_CM_SYNC_START_EVENT;
+import static com.sequenceiq.datalake.flow.datalake.cmsync.SdxCmSyncState.CORE_CM_SYNC_IN_PROGRESS_STATE;
+import static com.sequenceiq.datalake.flow.datalake.cmsync.SdxCmSyncState.CORE_CM_SYNC_STATE;
+import static com.sequenceiq.datalake.flow.datalake.cmsync.SdxCmSyncState.FINAL_STATE;
+import static com.sequenceiq.datalake.flow.datalake.cmsync.SdxCmSyncState.INIT_STATE;
+import static com.sequenceiq.datalake.flow.datalake.cmsync.SdxCmSyncState.SDX_CM_SYNC_FAILED_STATE;
+import static com.sequenceiq.datalake.flow.datalake.cmsync.SdxCmSyncState.SDX_CM_SYNC_FINISHED_STATE;
+
+import java.util.List;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.flow.core.config.AbstractFlowConfiguration;
+import com.sequenceiq.flow.core.config.AbstractFlowConfiguration.Transition.Builder;
+
+@Component
+public class SdxCmSyncFlowConfig extends AbstractFlowConfiguration<SdxCmSyncState, SdxCmSyncEvent> {
```

Comment:
It's not retryable. Currently it's not a problem since the user can just trigger it again, it doesn't do much.

Reply:
I did not change it. This will be invokeable from cdp-cli, so retry does not make much sense. If you like I can make it retryable.